pkg/v0/cmd.go

```diff
@@ -332,7 +336,7 @@ func cherryPick(ctx context.Context, logger *logrus.Entry, c internal.Commit, co
 		}
 	}
 
-	for _, cmd := range []*exec.Cmd{
+	verify := []*exec.Cmd{
```

Comment:
The `go mod` bits inside of the staging repo should remain on a per-commit basis.

Reply:
I'll look into that.

Reply:
That seems to work. The only thing that needs to be delayed is manifest generation, so a whole name has changed.